docs/en/introspection_endpoint.rst

```diff
@@ -84,11 +84,29 @@ The Introspection Endpoint responds with a JSON Object defined as follows.
 
    * - **Claim**
      - **Description**
-     - **Required**
+     - **Supported by**
    * - **active**
      - Boolean value that indicates the token validity. If the token is expired, it has been revoked or it
        has never been issued for the calling client_id, the Introspection Endpoint must return false.
-     - 
+     -  |spid-icon|
+   * - **scope**
+     - List of scopes required in the Authorization Request.
+     -  |spid-icon|
+   * - **exp**
+     - Token expiration.
+     -  |spid-icon|
+   * - **sub**
+     - Subject identifier. The same released in the ID Token. The RP MUST verify that the value is the same contained in the ID Token.
+     -  |spid-icon|
+   * - **client_id**
+     - URI of the RP registered in the federation. The RP MUST verify that the value is the same of the own client_id.
+     -  |spid-icon|
+   * - **iss**
+     - OP identified registered in the federation in Uniform Resource Locator (URL) format. The RP MUST verify that the value is the same of the OP queried.
```

Comment:
"OP identified registered" ...

cari sugli standard ufficiali abbiamo già le diciture in inglese di questi claims, consiglio di prendere da quelle

"identified registered" non è comprensibile in inglese

Reply:
Ho cercato di tradurre quanto scritto nelle LLGG, proponi testo idoneo

Reply:
siccome si tratta di SPID, eventualmente i colleghi lo sistemanio nelle successive PR per ora chiudo il commento